release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -56,6 +56,18 @@ link:https://access.redhat.com/support/policy/updates/openshift[Red Hat OpenShif
 
 This release adds improvements related to the following components and concepts.
 
+[id="ocp-4-6-web-console"]
+=== Web console
+
+[id="ocp-4-6-web-console-column-management"]
+==== Column management
+
+A *Manage columns* icon image:manage-columns.png[title="Manage Columns icon"] is
+now added to resources pages. When you click on the icon, all column names are
+listed with check boxes. Deselecting a check box will remove that column from
```

Comment:
default column names are listed on the left side of the modal, and additional column names are listed on the right.